buddyboss/buddyboss-platform
# Added filter for blog post feature media

### All Submissions:






### Changes proposed in this Pull Request:

Added below filter in the code and move feature image code in this filter.
add_filter( 'bb_add_feature_image_blog_post_as_avtivity_content', 'bb_add_feature_image_blog_post_as_avtivity_content_callback', 10, 2 );

Fixes #2516.

### How to test the changes in this Pull Request:

1. You need to check first with release branch and same testcases check with this PR. ( Not an issue just moved code in filter )
2. Settings -> Activity -> Post Type Comments -> Enable (Also select custom post type which you want to display in activity feed)
3. Create a diff blog post with long and short content and check in the activity feed
4. Just check ( I have just moved code in filter, not any issue in platform )

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/cr1F13V1l1a

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
